packages/@angular/cli/tasks/lint.ts

```diff
@@ -56,19 +54,37 @@ export default Task.extend({
           linter.lint(file, fileContents, configLoad.results);
         });
 
-        const result = linter.getResult();
-        errors += result.failureCount;
-        results = results.concat(result.output.trim().concat('\n'));
+        return linter.getResult();
+      })
+      .reduce((total, current) => {
+        const failures = current.failures
+          .filter((cf: any) => !total.failures.some((ef: any) => ef.equals(cf)));
+        total.failures = total.failures.concat(...failures);
+
+        if (current.fixes) {
+          total.fixes = (total.fixes || []).concat(...current.fixes);
+        }
+        return total;
```

Comment:
Originally had a one-liner but some of the built-in formatters expect `fixes` to be _falsy_ if none were made.